src/InsetOrderOptimizer.h

```diff
@@ -118,6 +118,31 @@ class InsetOrderOptimizer
      * closing that polyline into a polygon.
      */
     constexpr static coord_t coincident_point_distance = 10;
+
+    /*!
+     * Helper function to either iterate forward or backward over the path and output a vector of ExtrusionLines
+     *
+     * @tparam It an iterator type, forward or revers
+     * @param begin either the begin() or rbegin() of the path
+     * @param end either the end() or rend() of the path
+     * @return a vector of const ExtrusionLine pointers (whom ever came up with that container???)
+     */
+    template<typename It>
```

Comment:
WHy is this part of this PR?

Reply:
See comment above, this should not be part of this PR indeed